mypy/test/data.py

```diff
@@ -89,29 +89,34 @@ def parse_test_cases(
 
 
 class DataDrivenTestCase(TestCase):
+    name = None  # type: str
     input = None  # type: List[str]
     output = None  # type: List[str]
 
     file = ''
     line = 0
 
-    perform = None  # type: Callable[['DataDrivenTestCase'], None]
+    perform = None  # type: Callable[..., None]
```

Comment:
Is this meant to be the same signature as the `perform` parameter to `parse_test_cases` above?